prime-router/src/main/kotlin/azure/ReportFunction.kt

```diff
@@ -287,7 +287,7 @@ class ReportFunction(
         val generatedHashes = mutableListOf<String>()
         val duplicateIndexes = mutableListOf<Int>()
         for (rowNum in 0 until report.itemCount) {
-            var itemHash = report.getItemHashForRow(rowNum)
+            val itemHash = report.getItemHashForRow(rowNum)
```

Comment:
Can we remove all changes you made to this file?  it is not part of the mapping itself, and is going to cause conflicts with the work I'm doing.

Reply:
After our talk, you will just replace this file or merge its changes.  They are minor.